pkgs/by-name/po/porn-vault/package.nix

```diff
@@ -0,0 +1,103 @@
+{
+  fetchFromGitLab,
+  fetchurl,
+  rustPlatform,
+  lib,
+  pnpm,
+  stdenvNoCC,
+  nodejs_22,
+  ffmpeg,
+  imagemagick,
+  writeShellScript,
+  makeWrapper,
+  autoPatchelfHook,
+}:
+let
+  izzy = rustPlatform.buildRustPackage rec {
+    pname = "izzy";
+    version = "2.0.1";
+
+    src = fetchFromGitLab {
+      owner = "porn-vault";
+      repo = "izzy";
+      rev = version;
+      hash = "sha256-UauA5mZi5a5QF7d17pKSzvyaWbeSuFjBrXEAxR3wNkk=";
+    };
+
+    cargoHash = "sha256-y0bV83bnuSALwvRD8ztOiBE6LSg3Hx6HKSjvQPEkvIQ=";
+    cargoPatches = [ ./add-Cargo.lock.patch ];
+
+    meta = with lib; {
```

Comment:
Avoid with lib;

Reply:
Will do, honestly I hate the entire “with” thing on nix